src/orquestra/quantum/circuits/_matrices.py

```diff
@@ -219,3 +219,7 @@ def xy_matrix(angle):
             [0, 0, 0, 1],
         ]
     )
+
+
+def delay_matrix(_duration):
```

Comment:
Please add a comment to explain how this gate works (not sure where exactly TBH)

Reply:
Also, why it's `_duration` and not `_duration` ?

Reply:
The comment explaining how the gate works has been added. We use `_duration` instead of `duration` because it is a standard way of stressing down that a parameter is not used in the function. Reason why `_duration` is not used has also been explained.